src/repository/transform_photos.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import annotations
+
+from typing import Type
+
+import cloudinary
+from cloudinary import uploader
+from fastapi import HTTPException, status
+from sqlalchemy.orm import Session
+import uuid
+
+from src.conf.config import settings
+from src.database.models import Photo
+from src.repository.photos import get_photo_by_photo_id
+from src.schemas import TransformPhotoModel
+
+
+async def update_orig_photo_with_transformed_photo(
```

Comment:
Should be renamt with single underscore like "_update_orig_photo_with_transformed_photo".